ros/src/system/config/case/s1.yaml

```diff
@@ -7,7 +7,7 @@ case:
 
  actual:
   X0:
-   x: 200.0
+   x: 0.0
```

Comment:
do not change this. If you want create another case YAML file. You did not change any other the obstacle data and there are other demos that depend on this